src/main/java/frc/robot/utilities/TunableNumber.java

```diff
@@ -10,11 +16,16 @@
 import java.util.function.DoubleSupplier;
 
 public class TunableNumber extends SubsystemBase implements DoubleSupplier {
+  private NetworkTableInstance inst;
   private static ShuffleboardTab tuningTab;
   private GenericEntry numberEntry;
   private double value;
+  private double defaultValue;
```

Comment:
final?

Reply:
it gets made if the var is not inisialized when comp is on